synapse/lib/view.py

```diff
@@ -1315,6 +1315,14 @@ async def _calcChildViews(self):
 
                 todo.append(child)
 
+    def children(self):
```

Comment:
We should probably make this a generator as well with an asyncio sleep(0) in the skip loop in case we encounter instances with thousands of views.

Reply:
Sounds good, updated to a generator as well.